prow/plugins/help/help.go

```diff
@@ -51,13 +51,17 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The Config field is omitted because this plugin is not configurable.
-	return &pluginhelp.PluginHelp{
-			Description: "The help plugin provides commands that add or remove the '" + helpLabel + "' label from issues.",
-			WhoCanUse:   "Anyone can trigger this plugin on a PR.",
-			Usage:       "/[remove-]help",
-			Examples:    []string{"/help", "/remove-help"},
-		},
-		nil
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The help plugin provides commands that add or remove the '" + helpLabel + "' label from issues.",
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/[remove-]help",
+		Description: "Applies help-wanted label to an issue.",
```

Comment:
Applies or removes the label.
Also please use the `helpLabel` variable to compose the Description.

Reply:
done